tests/Unit/PointwiseFunctions/GeneralRelativity/WeylElectric.py

```diff
@@ -0,0 +1,14 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def weyl_electric_tensor(spatial_ricci, extrinsic_curvature,
+                         inverse_spatial_metric):
+    return (np.einsum("ij", spatial_ricci) +
+           np.einsum("kl,kl,ij", extrinsic_curvature,
+           inverse_spatial_metric, extrinsic_curvature) -
+           np.einsum("il,kl",
```

Comment:
can probably be written as `np.einsum("il,kl,kj", extrinsic_curvature, inverse_spatial_metric, extrinsic_curvature)`, and would also be more reliable (the product of those probably doesn't do what we want)